assets/css/oag.css

```diff
@@ -0,0 +1,208 @@
+                display: inline-block;
+                width: fit-content;
+            }
+        }
+        #wb-srch {
+            align-self: end;
+        }
+    }
+
+    h1#wb-cont, hgroup#wb-cont h1 {
+        border-bottom: .18em solid var(--primary-color);
+        -o-border-image: linear-gradient(to right, var(--primary-color) 71px,transparent 71px);
+        border-image: linear-gradient(to right,var(--primary-color) 71px,transparent 71px);
+        border-image-slice: 100%;
+        border-image-slice: 1;
+        border-left-width: 0;
+        border-right-width: 0;
+        border-top-width: 0;
+      }
+
+    .btn-primary {
+        background-color: var(--secondary-color);
+        border-color: #26374a;
+        border-radius: unset;
+
+        &:hover {
+            background-color: #1A303C;
+            border-color: #091c2d;
+        }
+    }
+
+    .alert-info {
+        border-color: var(--link-color);
+        -o-border-image: linear-gradient(to bottom, var(--link-color) 16px, var(--link-color) 16px, transparent 16px, transparent 48px, var(--link-color) 48px, var(--link-color) 48px) 1 100%;
+        border-image: linear-gradient(to bottom, var(--link-color) 16px, var(--link-color) 16px, transparent 16px, transparent 48px, var(--link-color) 48px, var(--link-color) 48px) 1 100%;
+        color: #676C72;
+
+        &::before {
+            color: var(--link-color);
+        }
+
+        & > :last-child {
+           padding-bottom: 0px;
+        }
+
+    }
+    .flex-container {
+        display: flex;              
+        flex-direction: column;    
+        justify-content: space-between; 
+        height: 100%;              
+        position: relative;
+        h3 {
+            width: fit-content;
+        }
+        .alert.alert-info {
+            margin-bottom: auto;       
+        }        
+    }   
+   
+    .btn-link {
+        color: var(--link-color);
+        background: linear-gradient(to right, var(--hover-color) 0%, var(--hover-color) 100%);
+        background-size: 0% 100%;
+        background-repeat: no-repeat;
+        transition: background-size 0.3s ease-in-out;
+        position: relative;
+        overflow: hidden;
+    
+        &:hover {
+            background-size: 100% 100%; 
+            color: var(--primary-color);
+            text-decoration: none;
+        }
+        &:not(:hover) {
+            background-size: 0% 100%;
+            transition: none;
+        }
+    }
+        
+    .label-primary {
+        background: #E3E0D3;
+        border-color: #908F84;
+        color: #4B5B62;
+    }
+
+    .text-primary {
+        color: var(--primary-color);
+    }
+
+    .well {
+        border: 2px solid #908F84;
+    }
+
+    .home {
+        .report {
+            section {
+                background-color: white;
+            }
+        }
+    }   
+
+    .panel-default {
+        border: 1px solid transparent;
+        background-color: #F4F4F2;
+    }
+    blockquote footer {
```

Comment:
Must be revised, not totally aligned with HTML spec.

> The [footer](https://html.spec.whatwg.org/multipage/sections.html#the-footer-element) element [represents](https://html.spec.whatwg.org/multipage/dom.html#represents) a footer for its nearest ancestor [sectioning content](https://html.spec.whatwg.org/multipage/dom.html#sectioning-content-2) element,

blockquote is not a site section element as per the spec: https://html.spec.whatwg.org/multipage/dom.html#sectioning-content but it does represent content block that are quoted